spec/requests/api/v1/targets/create_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe 'Api::V1::Targets', type: :request do
+  describe 'POST #create' do
```

Comment:
I think this should be something like this
`describe 'POST /api/v1/... ', type: :request do`